### PR TITLE
Change dynamic columns sample logic

### PR DESCRIPTION
**Pull request by AsafMah**
* Instead of having a percentage of the total, the user may decide on a maximum number of samples to get for buildschema.
* Null values will now not raise errors

@eladiw can you check I did k8s config ok?

### CODE REVIEW

File: charts/k2bridge/values.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ settings:
   instrumentationKey: "08f41044-7f85-4718-bd9b-51ecb72da320"
   collectTelemetry: false
   enableQueryLogging: true
+  maxDynamicSamples: null
```
Comment (eladiw) [inline charts/k2bridge/values.yaml:27]:
```suggestion
  # maxDynamicSamples: <number>
```

**Review by eladiw — APPROVED** [review]
lgtm

**Review by meguittet — APPROVED** [review]

**Review by tomconte — APPROVED** [review]
LGTM

File: K2Bridge.Tests.UnitTests/KustoDAL/KustoDataAccessTests.cs [hunk 2]
```diff
@@ -598,7 +598,7 @@ public async Task GetFieldCaps_WithDynamicColumnArray_ReturnFieldCaps()
         [Test]
         public async Task GetFieldCaps_WithPercentage_ReturnCorrectQuery()
```
Comment (dupuyjs) [inline K2Bridge.Tests.UnitTests/KustoDAL/KustoDataAccessTests.cs:599]: Suggest to change the name GetFieldCaps_WithSamples_ReturnCorrectQuery.

**Pull request merged**